src/expr-test-util/tests/test_runner.rs

```diff
@@ -91,6 +91,7 @@ mod test {
                         }
                     }
                     "rel-to-test" => {
+                        println!("nikhil {}", s.input);
```

Comment:
I think maybe you left this in by mistake?

Reply:
Ugh, yes, thanks. Tracking that one down was miserable.